### PR TITLE
Add auto-correct to DoubleStartEndWith

### PR DESCRIPTION
**Pull request by rrosenblum**

### CODE REVIEW

File: lib/rubocop/cop/performance/double_start_end_with.rb [hunk 1]
```diff
@@ -56,13 +70,11 @@ def combine_args(first_call_args, second_call_args)
...
           add_offense(node,
                       :expression,
-                      format(
-                        MSG,
-                        receiver: receiver.source,
-                        method: method,
-                        combined_args: combined_args,
-                        original_code: node.source
-                      ))
+                      format(MSG,
```
Comment (bbatsov) [inline lib/rubocop/cop/performance/double_start_end_with.rb]: Changing the formatting here seems unrelated to the purpose of the PR.

Comment (rrosenblum) [inline lib/rubocop/cop/performance/double_start_end_with.rb]: I have updated this to not modify the format here.

**Comment by bbatsov** [comment]
👍
